.github/workflows/wcf_backport_dispatch.yml

```diff
@@ -0,0 +1,181 @@
+on:
+  workflow_call:
+    inputs:
+      pr_number:
+        description: "number of the PR to be backported"
+        type: string
+        default: ""
+        required: true
+      target_branches:
+        description: "target branches to backport to"
+        type: string
+        default: ""
+        required: false
+      label:
+        description: "The label to use to indicate backport"
+        type: string
+        default: "backport"
+        required: false
+      branchFilePath:
+        description: "A file that contains the list of active branches"
+        type: string
+        default: "active-branches.json"
+        required: false
+      gitUser:
+        description: "The user to use for commit"
+        type: string
+        default: "kumahq[bot]"
+        required: false
+      gitEmail:
+        description: "The user to use for commit"
+        type: string
+        default: "<110050114+kumahq[bot]@users.noreply.github.com>"
+        required: false
+    secrets:
+      app_id:
+        required: false
+      private_key:
+        required: false
+env:
+  GH_REPO: ${{ github.repository }}
+jobs:
+  collect-info:
+    outputs:
+      pr_title: ${{ steps.get-pr-info.outputs.pr_title }}
+      pr_state: ${{ steps.get-pr-info.outputs.pr_state }}
+      pr_base_ref: ${{ steps.get-pr-info.outputs.pr_base_ref }}
+      pr_merge_commit_sha: ${{ steps.get-pr-info.outputs.pr_merge_commit_sha }}
+      branches: ${{ steps.generate-matrix.outputs.branches }}
+    runs-on: ubuntu-latest
+    steps:
+      - id: get-pr-info
+        name: get-pr-info
+        env:
+          GH_TOKEN: ${{ github.token }}
+          GH_DEBUG: ${{ runner.debug == '1' }}
+        run: |
+          if [[ "${{ runner.debug }}" == "1" ]]; then
+            set -x
+          fi
+
+          PR_INFO_JSON=$(gh pr view ${{ inputs.pr_number }} --json 'number,title,mergedAt,state,mergeCommit,baseRefName' || echo '{}')
```

Comment:
fetch PR info using the `gh` CLI, to replace getting the information from the event context from the orginal version of the workflow